packages/analytics-js/src/services/StoreManager/storages/CookieStorage.ts

```diff
@@ -39,6 +40,8 @@ class CookieStorage implements IStorage {
     }
     this.isSupportAvailable = isStorageAvailable(COOKIE_STORAGE, this, this.logger);
     this.isEnabled = Boolean(this.options.enabled && this.isSupportAvailable);
+    delete this.options.enabled;
+    state.storage.cookie.value = this.options;
```

Comment:
Let's move this to the `storageEngine` module where we call the `configure` method. This is too deep to introduce the `state` dependency.